meta-intel-edison-bsp/recipes-kernel/linux/linux-yocto_6.0.bb

```diff
@@ -0,0 +1,103 @@
+KBRANCH ?= "master"
+
+require recipes-kernel/linux/linux-yocto.inc
+
+LICENSE = "GPLv2"
+LIC_FILES_CHKSUM = "file://COPYING;md5=6bc538ed5bd9a7fc9398086aedcd7e46"
+
+SRC_URI = "git://github.com/torvalds/linux.git;protocol=https"
+
+# based on andy-shev's edison kernel configs https://github.com/andy-shev/linux/commits/eds-acpi
+SRC_URI:append = " file://0001-enable-to-build-a-netboot-image.cfg"
+SRC_URI:append = " file://0002-enable-x2APIC.cfg"
+SRC_URI:append = " file://0003-enable-NETCONSOLE_DYNAMIC.cfg"
+SRC_URI:append = " file://0004-enable-STMMAC.cfg"
+SRC_URI:append = " file://0005-enable-IGB.cfg"
+SRC_URI:append = " file://0006-enable-IXGBE.cfg"
+SRC_URI:append = " file://0007-enable-USB_RTL8152.cfg"
+SRC_URI:append = " file://0008-disable-HPET.cfg"
+SRC_URI:append = " file://0009-disable-i915-DRM.cfg"
+SRC_URI:append = " file://0010-enable-X86_INTEL_MID.cfg"
+SRC_URI:append = " file://0011-enable-INPUT_SOC_BUTTON_ARRAY.cfg"
+SRC_URI:append = " file://0012-enable-I2C_HID-and-HID_MULTITOUCH.cfg"
+SRC_URI:append = " file://0013-DEBUG_SHIRQ-DEBUG_LOCKDEP.cfg"
+SRC_URI:append = " file://0014-enable-ACPI_DEBUG-and-ACPI_PROCFS_POWER.cfg"
+SRC_URI:append = " file://0015-enable-ACPI_TABLE_UPGRADE.cfg"
+SRC_URI:append = " file://0016-enable-X86_INTEL_LPSS-and-LPSS-drivers.cfg"
+SRC_URI:append = " file://0017-enable-MFD_INTEL_LPSS-drivers.cfg"
+SRC_URI:append = " file://0018-enable-INTEL_IDMA64-iDMA-64-bit.cfg"
+SRC_URI:append = " file://0020-enable-SPI_DW.cfg"
+SRC_URI:append = " file://0021-disable-HDA-audio.cfg"
+SRC_URI:append = " file://0022-enable-Intel-Quark-devices.cfg"
+SRC_URI:append = " file://0023-enable-DEBUG_GPIO.cfg"
+SRC_URI:append = " file://0024-enable-GPIO_DWAPB.cfg"
+SRC_URI:append = " file://0025-enable-GPIO_PCA953X.cfg"
+SRC_URI:append = " file://0029-enable-INTEL_IDLE.cfg"
+SRC_URI:append = " file://0030-enable-PUNIT_ATOM_DEBUG.cfg"
+SRC_URI:append = " file://0031-enable-REGULATOR.cfg"
+SRC_URI:append = " file://0032-enable-BRCMFMAC.cfg"
+SRC_URI:append = " file://0033-enable-BT_HCIUART_BCM.cfg"
+SRC_URI:append = " file://0034-enable-ADS7950.cfg"
+SRC_URI:append = " file://0035-enable-ACPI_CONFIGFS.cfg"
+SRC_URI:append = " file://0036-enable-SND_SST_ATOM_HIFI2_PLATFORM.cfg"
+SRC_URI:append = " file://0037-enable-SND_SOC_SOF-nocodec.cfg"
+SRC_URI:append = " file://0038-enable-PHY_TUSB1210.cfg"
+SRC_URI:append = " file://0039-enable-USB_CONFIGFS.cfg"
+SRC_URI:append = " file://0040-enable-INTEL_MRFLD_ADC.cfg"
+SRC_URI:append = " file://0041-enable-EXTCON_INTEL_MRFLD.cfg"
+# FIXME: when building 5.13 and above for 32b the stack protector code hangs
+SRC_URI:append = " ${@bb.utils.contains("DEFAULTTUNE", "corei7-32", " file://stack.cfg", "", d)}"
+
+# our additional configs
+SRC_URI:append = " file://ftdi_sio.cfg"
+SRC_URI:append = " file://ch341.cfg"
```

Comment:
Are you going to include the series that enables in particular GPIO on CH341 compatible chips?